### ISSUE TITLE
Invalid calibration_points for roidmi.vacuum.v60

### ISSUE THREAD

**Issue by wowgamr** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#193]
### Checklist

- [X] I have updated the card to the latest version available
- [X] I have cleared the cache of my browser
- [X] I have checked if the problem is already reported

### The problem

Can't select zone for cleaning because selector locked in the upper left corner.
![image](https://user-images.githubusercontent.com/4518407/142871628-2b12afd9-d0f9-4aba-969c-19dbd89f3a77.png)

### What version of a card has described problem?

v2.0.3

### What vacuum model do you have problems with?

roidmi.vacuum.v60

### Which integration do you use to control your vacuum (link)?

https://github.com/al-one/hass-xiaomi-miot

### What browser (browsers/apps) does have this problem?

Chrome 95.0.4638.69

### What version of Home Assistant do you use?

core-2021.11.3

### What type of installation are you running?

Home Assistant Supervised

### Card's configuration

```yaml
- type: custom:xiaomi-vacuum-map-card
    entity: vacuum.roidmi_v60_9cd4_robot_cleaner
    map_source:
      camera: camera.xiaomi_cloud_map_extractor
    map_locked: true
    calibration_source:
      camera: true
    icons: []
    tiles: []
    map_modes:
      - template: vacuum_clean_zone
```

### Javascript errors shown in the browser's console (if applicable)

```shell
Error: <polygon> attribute points: Expected number, "NaN, NaN NaN, Na…".
W @ xiaomi-vacuum-map-card.js:31
Error: <polygon> attribute points: Expected number, "NaN, NaN NaN, Na…".
_setup @ xiaomi-vacuum-map-card.js:197
```

### Additional information

I tried to disable map rotation, but it didn't help me.
![image](https://user-images.githubusercontent.com/4518407/142872579-c3c7e3b1-4946-4b90-b75a-c08e1afe5eaa.png)

**Comment by PiotrMachowski** [comment]
Problem should be solved in v2.1.2

**Comment by wowgamr** [comment]
Thank you! It works fine.
Happy New Year!
